doc/FAQ.rst

```diff
@@ -4,6 +4,117 @@ FAQ
 Here is a list of questions we have either been asked by users or
 potential pitfalls we hope to help users avoid:
 
+**Q: How do I adjust the x/y/z axis bounds (matplotlib's xlim, ylim)?**
+
+**A:** Pass an unpacked dictionary containing the kdims/vdims' names as
+keys and a tuple of the bounds as values into obj.redim.range().
+
+This constrains the bounds of x_col to (0, max(x_col)).
+
+.. code:: python
+
+  curve = hv.Curve(df, 'x_col', 'y_col')
+  curve = curve.redim.range(x_col=(0, None), z_col=(None, 10))
+
+If your column names have spaces, you may predefine a dictionary
+and unpack it in obj.redim.range(). This same method is applicable to
+adjust the range of a color bar.
+
+.. code:: python
+
+  bounds = {'x col': (0, None), 'z col': (None, 10)}
+  curve = hv.Curve(df, 'x col', ['y col', 'z col']).options(color_index='z col')
+  curve = curve.redim.range(**bounds)
+
+**Q: How do I export a figure?**
+
+**A:** Create a renderer object by passing a backend (matplotlib / bokeh)
+and pass the object and name of file without any suffix into the .save method.
+
+.. code:: python
+
+  backend = 'bokeh'
+  renderer = hv.renderer(backend)
+  renderer.save(obj, 'name_of_file')
+
+Just note that if have used Jupyter magic (%%opts) to customize your plot
+and you try to export it, the customizations will not be retained in the export.
+
+.. code:: python
+
+  %%opts Curve [width=1000]
+  curve = hv.Curve([1, 2, 3])
+  hv.renderer('bokeh').save(curve, 'example_curve')  # width will not be saved
+
+You'll have to pass the keyword arguments into .options().
+
+.. code:: python
+
+  curve = hv.Curve([1, 2, 3]).options(width=1000)
+  hv.renderer('bokeh').save(curve, 'example_curve')
+
+**Q: How do I provide axes labels?**
+
+**A:** Pass a tuple containing the column name and label.
```

Comment:
"A: One convenient way is to pass a tuple"... (it's not *the* way, just one way).